styles.css

```diff
@@ -5,24 +5,30 @@ body {
   font-family: "nunito", sans-serif;
   background-color: #eceef1;
 }
+
 .nav-container {
   display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
for the first problem we were told to do a mint.com sign up page replica and it had an extra line >(including mint) so that's why learn more was on another line.
as for the other problems I have solved them as requested.